packages/api-plugin-promotions-coupons/migrations/2.js

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable no-await-in-loop */
+import Random from "@reactioncommerce/random";
+import getCurrentShopTime from "./getCurrentShopTime.js";
+
+/**
+ * @summary returns an auto-incrementing integer id for a specific entity
+ * @param {Object} db - The db instance
+ * @param {String} shopId - The shop ID
+ * @param {String} entity - The entity (normally a collection) that you are tracking the ID for
+ * @return {Promise<Number>} - The auto-incrementing ID to use
+ */
+async function incrementSequence(db, shopId, entity) {
```

Comment:
We have a function defined in the code already for this, we should not duplicate it here.